src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/RecommendCaseInsensitiveStringComparison.Fixer.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.NetCore.Analyzers.Performance
+{
+    /// <summary>
+    /// CA1862: Prefer the StringComparison method overloads to perform case-insensitive string comparisons.
+    /// </summary>
+    public class RecommendCaseInsensitiveStringComparisonFixer : CodeFixProvider
```

Comment:
Missing ExportCodeFixProvider and Shared attributes.
Consider sealing as well.

Reply:
Good catches. I was originally assuming I would need this to be abstract so that I could create a C# and a VB implementation, but I didn't need them in the end (so far). But forgot to add the attribute here.

Since I still have some TODOs left, I wouldn't be surprised if I ended up needing the C# and VB classes in the end. For now, I'll address your suggestion.